app/models/redis_store.rb

```diff
@@ -2,12 +2,24 @@
 class RedisStore
   extend ActiveModel::Naming
   extend ActiveModel::Callbacks
-  define_model_callbacks :initialize, only: :after
 
   include ActiveModel::Serialization
   include ActiveModel::Validations
   include Virtus.model
-  REDIS_STORE = Redis.current
+
+  define_model_callbacks :initialize, only: :after
+
+  class << self
+    attr_accessor :redis, :ttl
+  end
+
+  def self.redis_store(namespace)
+    @redis ||= Redis::Namespace.new(namespace, redis: Redis.current)
+  end
+
+  def self.default_ttl(ttl)
+    @ttl ||= ttl
+  end
```

Comment:
Should we possibly also make redis_key a class level accessor that is set so that we don't need to use alias? you can make the setter take the method name, and then an instance level attribute reader that references it.